README.md

```diff
@@ -506,6 +543,46 @@ $ rustup target add thumbv7em-none-eabihf
 $ ./test.sh
 ```
 
+## API
+
+See the Tracing section of the [Modality Instrumentation SDK](https://docs.auxon.io/modality/reference/c-api.html) for detailed documentation. 
+Note that the Mutation section of the [Instrumentation SDK](https://docs.auxon.io/modality/reference/c-api.html) reflects functionality exclusive to commercial Modality. 
+
+See the comments in [probe.h](./modality-probe-capi/include/probe.h) for local documentation.
+
+## CMake
+
+The release package provides a find script and a set of CLI helper functions
```

Comment:
Would be good to have Jon sanity-check this, I don't speak cmake

Reply:
@jonlamb-gh ^

Reply:
looks right :+1: